quickwit/quickwit-metastore/src/metastore/postgresql_metastore.rs

```diff
@@ -1895,4 +1969,51 @@ mod tests {
              expression: `^[a-zA-Z\\*][a-zA-Z0-9-_\\.\\*]{0,254}$``"
         );
     }
+
+    #[sqlx::test(migrator = "MIGRATOR")]
+    async fn test_shards_table_empty(pool: Pool<Postgres>) -> sqlx::Result<()> {
+        let pg_shard: Option<PgShard> = sqlx::query_as::<_, PgShard>("SELECT * FROM shards")
+            .fetch_optional(&pool)
+            .await?;
+
+        assert!(pg_shard.is_none());
+
+        Ok(())
+    }
+
+    #[sqlx::test(migrator = "MIGRATOR")]
```

Comment:
@guilload This is not the complete series of test but this a good first test to check if `sqlx::test` works and I feel it does :) . If you happy with this I can add these to the main tests in the other directory.

Reply:
Let's focus on adding tests to the metastore shard testsuite in `src/tests/shard.rs` and only add tests specific to the Pg implementation here.